src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable jsx-a11y/role-supports-aria-props */
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useState } from 'react'
+
+export interface AccordionProps {
+    type: 'base' | 'single'
```

Comment:
Make type optional. 
If the user don't pass the type, should render the base one :)

Reply:
Done, fixed